### PR TITLE
Update to use new documentation template.

### PR DESCRIPTION
**Pull request by vlsunil**
Signed-off-by: Sunil V L <sunilvl@ventanamicro.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,36 +1,21 @@
...
-DITAA = ditaa
-RISCV_ACPI_SPEC = riscv-acpi-platform-req
-PANDOC = pandoc
+all: build
 
-# Build the platform spec in several formats
-all: $(RISCV_ACPI_SPEC).md $(RISCV_ACPI_SPEC).pdf $(RISCV_ACPI_SPEC).html
+build:
 
-$(RISCV_ACPI_SPEC).md: $(RISCV_ACPI_SPEC).xml
```
Comment (jjscheel) [inline Makefile:13]: The RISC-V structure is only for building PDFs.  As such, building XML, HTML, and other targets need not be removed.  Support and/or removal is a community decision.

File: header.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+:data-uri:
+:hide-uri-scheme:
+:stem: latexmath
+:footnote:
+:xrefstyle: short
+
+= RISC-V ACPI Platform Requirements Specification
+Sunil V L; RISC-V Platform HSC Group
+
+// Preable
```
Comment (jjscheel) [inline header.adoc]: Oops, looks like we have a template problem: "Preable" -> "Preamble"

**Review by jjscheel — APPROVED** [review]
Just a couple things to consider.

PDF looks good too.  

Nice work!  THANK YOU!!!

File: Makefile (hunk 1 above)
Comment (vlsunil) [reply Makefile:13]: Thanks. This was carried from another makefile template. PDF should be sufficient for now.

File: header.adoc (hunk 2 above)
Comment (vlsunil) [reply header.adoc]: Thanks will fix it.

**Pull request merged**